### ISSUE TITLE
Test with DQN tensorflow version

### ISSUE THREAD

**Issue by tohnperfect** [nadavbh12/Retro-Learning-Environment#12]
I try to modify the tensorflow DQN ALE version [this repo](https://github.com/gtoubassi/dqn-atari) to use RLE instead of ALE. However, I got the Segmentation fault error as follows,

> I tensorflow/stream_executor/dso_loader.cc:128] successfully opened CUDA library libcublas.so locally
> I tensorflow/stream_executor/dso_loader.cc:128] successfully opened CUDA library libcudnn.so locally
> I tensorflow/stream_executor/dso_loader.cc:128] successfully opened CUDA library libcufft.so locally
> I tensorflow/stream_executor/dso_loader.cc:128] successfully opened CUDA library libcuda.so.1 locally
> I tensorflow/stream_executor/dso_loader.cc:128] successfully opened CUDA library libcurand.so locally
> Arguments: Namespace(compress_replay=False, core='./snes9x2010_libretro.so', eval_epoch_steps=125000, learning_rate=0.00025, model=None, normalize_weights=True, observation_steps=50000, prioritized_replay=False, replay_capacity=1000000, rom='./mortal_kombat.sfc', save_model_freq=10000, screen_capture_freq=250, target_model_update_freq=10000, train_epoch_steps=250000)
> Warning: couldn't load settings file: 
> R.L.E: Retro Learning Environment (version 1.0.0)
> [Based upon the Arcade Learning Environment (A.L.E)]
> [Powered by LibRetro]
> Use -help for help screen.
> [inf] Frontend supports RGB565 - will use that instead of XRGB1555.
> Sound buffer size: 128000 (32000 samples)
> Core loaded
> [inf] No ROM file header found.
> Map_LoROMMap
> PPU.RenderSub = 0
> PPU.FullClipping = 1
> Settings.Transparency = 1
> Settings.SpeedhackGameID = 0
> PPU.SFXSpeedupHack = 0
> coldata_update_screen = 1
> [inf] "MORTAL KOMBAT" [checksum ok] LoROM, 16Mbits, ROM, NTSC, SRAM:0Kbits, ID:____, CRC32:DEF42945
> Running ROM file...
> Random seed is 65
> Segmentation fault (core dumped)
> 

I did rename the rom file to agree with the name in the [header file](https://github.com/nadavbh12/Retro-Learning-Environment/blob/master/src/games/supported/MortalKombat.hpp).

Please give me some advices?

**Comment by nadavbh12** [comment]
hey @tohnperfect,
Looks like there was a bug in the getRGB function.
In addition to RGB, a fourth, currently unused, channel is returned for the alpha.
It should work correctly now.
